atoma-proxy/src/server/mod.rs

```diff
@@ -26,6 +27,8 @@ use crate::sui::Sui;
 const CHAT_COMPLETIONS_PATH: &str = "/v1/chat/completions";
 const MODELS_PATH: &str = "/v1/models";
 const NODE_PUBLIC_ADDRESS_REGISTRATION: &str = "/node/registration";
+const STACK_ENTRY_COMPUTE_UNITS: u64 = 1000;
```

Comment:
Can you add comments for all these constants ? The ones above, this one, and the ones below

Reply:
Added todos